packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -3,46 +3,51 @@
         :class="[
             $style['c-filterPill'], {
                 [$style['c-filterPill--selected']]: isToggleSelected,
-                [$style['c-filterPill--disabled']]: isToggleDisabled
+                [$style['c-filterPill--disabled']]: isDisabled
```

Comment:
Do we want to change this as the opposite state is `isToggleSelected`?
should we align? or not change?

Reply:
I think if there are more improvements to be made here then it would be best suited to a separate PR. As I understand, the `isDisabled` prop is to prevent the component from being interacted with/greyed out